HealthcareSystem/Backend/Service/NotificationSurveyAndFeedback/FeedbackService.cs

```diff
@@ -13,22 +14,33 @@ namespace Service.NotificationSurveyAndFeedback
 {
    public class FeedbackService
    {
-        private FeedbackRepository feedbackRepository = new FeedbackRepository();
-        public void NewFeedback(Feedback feedback)
+        private IFeedbackRepository _feedbackRepository;
+
+        private ActivePatientRepository _activePatientRepository = new ActivePatientRepository();
```

Comment:
Since you're using the ActivePatientRepository, maybe make an interface there as well now? That way you won't have to touch this class again when that part of the system is refactored. Also, just a style suggestion, but I would remove the empty line between the two fields.